docs/individual_sheets.md

```diff
@@ -75,16 +66,16 @@ Define all materials that will be referenced throughout the document.
 > and <a href="https://criptapp.org/keys/material-property-key/" target="_blank">material property</a> columns as you needed
 
 | Row 2                                                                                                                            | Row 1      | Required | Row 5 - ∞ expected value |
-|----------------------------------------------------------------------------------------------------------------------------------|------------|----------|--------------------------|
+| -------------------------------------------------------------------------------------------------------------------------------- | ---------- | -------- | ------------------------ |
+| inventory                                                                                                                        | relation   | no       | inventory name           |
 | \*name                                                                                                                           | attribute  | yes      | unique name              |
 | pick from `Name` column of <a href="https://criptapp.org/keys/material-identifier-key/" target="_blank">material identifiers</a> | identifier | no       | your values              |
 | pick from `Name` column of <a href="https://criptapp.org/keys/material-property-key/" target="_blank">material properties</a>    | property   | no       | your values              |
-| use_existing                                                                                                                  | property   | no       | TRUE, FALSE, or blank    |
+| \*use_existing                                                                                                                   | property   | yes      | project name or FALSE    |
```

Comment:
I think this is great currently and it is enough to get the job done.

I thought we made a feature that it would be `project name` and we would `use_existing` and if it is `False` or empty then we wouldn't?  Are we still thinking of adding that feature later? Or am I getting it fully wrong?

I'm thinking if the user has to write a 100 rows of `False` they'll get angry with us very quickly